src/seaport.ts

```diff
@@ -349,15 +354,14 @@ export class Seaport {
     counter: number,
     accountAddress?: string
   ): Promise<string> {
-    const signer = this.provider.getSigner(accountAddress);
+    const signer = this.signer ?? this.provider.getSigner(accountAddress);
```

Comment:
can we make this into a method that's just `_getSigner(accountAddress?: str)` that will return `this.signer` if signer is defined, otherwise fallback to the provider's `getSigner(accountAddress)` method.